tools/accuracy_checker/dataset_definitions.yml

```diff
@@ -1142,3 +1142,11 @@ datasets:
       prefix: duration_prediction
       input_suffix: in
       reference_suffix: out
+
+  - name: GoPro
+    data_source: GOPRO/test
+    annotation_conversion:
+      converter: image_processing
+      data_dir: GOPRO/test
+      input_suffix: blur.png
+      target_suffix: sharp.png
```

Comment:
please add annotation pickle (required for testing)

Reply:
done